config/runtime.exs

```diff
@@ -137,11 +137,14 @@ end
 ###################################
 # App configuration: game_backend #
 ###################################
-{:ok, currency_config_json} =
-  Application.app_dir(:game_backend, "priv/currencies_rules.json")
-  |> File.read()
 
-config :game_backend, :currencies_config, Jason.decode!(currency_config_json)
+if System.get_env("RELEASE") == :central_backend or config_env() == :dev do
```

Comment:
`:central_backend` should be an string, shouldn't it?

Reply:
you're right! the deploy didn't failed because it didn't ran at all :P